sample/index.html

```diff
@@ -50,15 +53,17 @@
 		will be working on in the next few iterations of the project. Stay
 		tuned for some updates soon!</p>
 </div>
+
 <script src="../build/ckeditor.js"></script>
 <script>
 	ClassicEditor.create( document.querySelector( '#editor' ) )
-		.then( editor => {
-			window.editor = editor;
-		} )
-		.catch( err => {
-			console.error( err.stack );
-		} );
+	.then( editor => {
```

Comment:
I wasn't right. I thought about such a case:

```js
ClassicEditor.create( document.querySelector( '#editor' ), {
	plugins: [ EssentialsPreset, Paragraph, Bold, Italic, Heading ],
	toolbar: [ 'headings', 'bold', 'italic', 'undo', 'redo' ]
} )
.then( editor => {
	window.editor = editor;
} )
.catch( err => {
	console.error( err.stack );
} );
```

But here we have `} )` ending with indent 0, so that's why we continue with such an indentation. 

In this case where `.then()` follows an expression which finished in one line we should rather indent them, just like @pomek did.